src/invision/components/movie-item/movie-item.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './movie-item.scss';
+import MoviePopup from '../movie-popup/movie.popup';
+import DeletePopup from '../delete-movie/delete-popup';
+
+
+export default class MovieItem extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      isEditMovieVisible: false,
+      isMenuVisible: false,
+      isDeletePopup: false,
+    }
+    this.showMenu = () => {
+      this.setState({ isMenuVisible: 'block' });
+    }
+
+    this.hideMenu = () => {
+      this.setState({ isMenuVisible: 'none' });
+    }
+
+    this.onModalClose = this.onModalClose.bind(this);
+    this.onDeletePopupClose = this.onDeletePopupClose.bind(this);
+  }
+
+  editMovie() {
+    this.setState({ isEditMovieVisible: true });
+  }
+
+  onModalClose(visible) {
```

Comment:
It is very strange to use variable from the child component to close popup because it can return any value. You should provide only callback to the child component and set **isEditMovieVisible** to false there